synapse/lib/stormlib/cell.py

```diff
@@ -210,30 +211,35 @@ async def _hotFixesCheck(self):
 
         return dowork
 
+    @s_stormtypes.stormfunc(readonly=True)
     async def _getCellInfo(self):
         if not self.runt.isAdmin():
             mesg = '$lib.cell.getCellInfo() requires admin privs.'
             raise s_exc.AuthDeny(mesg=mesg, user=self.runt.user.iden, username=self.runt.user.name)
         return await self.runt.snap.core.getCellInfo()
 
+    @s_stormtypes.stormfunc(readonly=True)
```

Comment:
Is there any concern about returning admin restricted info even if it doesn't modify anything?

Reply:
I'm not aware of any concerns there. An unprivileged execution context would still not be able to access this information.

One scenario where this could be used is in an extended HTTP API handler. A handler setup to run as the owner ( an admin ) in readonly mode could then access this information and that should be okay IMO; since that would be an opt-in behavior by the org/individuals configuring their Cortex.